packages/waku/src/router/define-router.tsx

```diff
@@ -46,6 +46,15 @@ const safeJsonParse = (str: unknown) => {
   return undefined;
 };
 
+export function DefaultRoot({ children }: { children: ReactNode }) {
+  return (
+    <html>
+      <head></head>
+      <body>HELLO WORLD{children}</body>
+    </html>
+  );
+}
```

Comment:
@dai-shi I am pretty certain putting this here is wrong, but at the same time I am unsure where it should go or how it should be written.

Should I be trying to create this with react's `createElement` somehow instead?

Reply:
I think it should be in `create-pages.ts`. Yes, use `createElement`.

Reply:
great! thanks, so the very basic example works. Now I'll work on supporting `_root.tsx`, then updating the examples & tests :)

Reply:
that's quick!